cmd/snap/cmd_export_key_test.go

```diff
@@ -48,6 +49,33 @@ func (s *SnapKeysSuite) TestExportKey(c *C) {
 	c.Assert(rest, DeepEquals, []string{})
 	pubKey, err := asserts.DecodePublicKey(s.stdout.Bytes())
 	c.Assert(err, IsNil)
-	c.Check(pubKey.ID(), Equals, "2uDFKgzxAPJ4takHsVbPFjmszLvaxg431C1KmhKFPwcD96MLKWcKj9cFEePrAZRs")
+	c.Check(pubKey.ID(), Equals, "g4Pks54W_US4pZuxhgG_RHNAf_UeZBBuZyGRLLmMj1Do3GkE_r_5A5BFjx24ZwVJ")
 	c.Check(s.Stderr(), Equals, "")
 }
+
+func (s *SnapKeysSuite) TestExportKeyAccount(c *C) {
+	rootPrivKey, _ := assertstest.GenerateKey(1024)
+	storePrivKey, _ := assertstest.GenerateKey(752)
+	storeSigning := assertstest.NewStoreStack("canonical", rootPrivKey, storePrivKey)
+	db, err := asserts.OpenDatabase(&asserts.DatabaseConfig{
```

Comment:
not sure why the full db here? wouldn't just the KeypairManager be enough? not a blocker though